crowbar_framework/app/controllers/api/upgrade_controller.rb

```diff
@@ -540,4 +541,17 @@ def dummy_crowbar_init_api_upgrade_connect
   def backup_params
     params.require(:backup).permit(:name)
   end
+
+  def async_save_fail
+    error = Api::Error.find_by(error: "Crowbar::Error::SaveUpgradeStatusError")
+    return unless error
+    render json: {
+      errors: {
+        error.caller => {
+          data: error.message,
+          help: I18n.t("api.upgrade.#{error.caller}.help.default")
```

Comment:
Hm, default message for a step will not be helpful here, I'm afraid. I think we need to advice to do something about the progress file.